backend/api/view/AuthorProfileView.py

```diff
@@ -13,28 +19,29 @@ class AuthorProfileView(generics.GenericAPIView):
 
     def is_mutable(self, key):
         for ele in self.mutable_keys:
-            if(key == ele):
+            if (key == ele):
```

Comment:
You can probably just check if the key is in self.mutable_keys instead. Same thing as you're doing here but a nice one liner

Reply:
you wrote this... i didnt change anything.......